config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -13,6 +13,6 @@ periodics:
     path_alias: "sigs.k8s.io/cluster-api-provider-aws"
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190315-49d62eb51-1.12
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190316-dae90a14f-1.12
       command:
-      - "./scripts/ci-aws-cred-test.sh"
```

Comment:
Is this an intentional change?

Reply:
addition of a newline from `sed` or smth?

Reply:
Seems like a behavior of BSD "sed" on Mac. 
https://stackoverflow.com/questions/13325138/why-does-sed-add-a-new-line-in-osx
So the answer to my question is its unintentional and benign change. Infact we might appreciate the addition of the "newline" char